tests/neptune/new/test_run.py

```diff
@@ -122,6 +122,13 @@ def test_assign_false(self):
 
         self.assertFalse(exp["params/predictor.cheat"].fetch())
 
+    def test_assign_copy(self):
+        exp = init(mode="debug")
+        exp["some/path/num"] = 42
+        exp["copied/path"] = exp["some/path/num"]
+        exp.wait()
+        self.assertEqual(42, exp["copied/path"].fetch())
+
```

Comment:
yeah, that was really broken... fixed now, I hope -- thanks!

Reply:
Have I broken this because String does not support copying yet?

```
        exp["some/path/num"] = "fo9o"
        exp["copied/path"] = 42
        exp["copied/path"] = exp["some/path/num"]
```

```
self = <neptune.new.internal.backends.neptune_backend_mock.NeptuneBackendMock.NewValueOpVisitor object at 0x7f0ef5fe2670>
op = CopyAttribute(path=['copied', 'path'], container_id='bdcc8dc7-a2b7-42a9-ba25-209a43888ed7', source_path=['some', 'path', 'num'], source_attr_cls=<class 'neptune.new.attributes.atoms.string.String'>)

    def visit_copy_attribute(self, op: CopyAttribute) -> Optional[Value]:
>       getter = op.source_attr_cls.getter
E       AttributeError: type object 'String' has no attribute 'getter'
```


Additionally it'd be good to have additional case with copying on existing value (the case recently broken).

Reply:
> Have I broken this because String does not support copying yet?
yep
> Additionally it'd be good to have additional case with copying on existing value
will do